components/notifier/tests/notifier_utilities/test_data.py

```diff
@@ -26,37 +26,52 @@ class NotifyUtilitiesTests(unittest.TestCase):
     def setUp(self) -> None:
         """Define info that is used in multiple tests."""
         self.client = mongomock.MongoClient()
-        self.now = datetime.now().replace(microsecond=0, tzinfo=timezone.utc).isoformat()
-        self.just_now = (datetime.now().replace(microsecond=0, tzinfo=timezone.utc) - timedelta(hours=1)).isoformat()
-        self.older = (datetime.now().replace(microsecond=0, tzinfo=timezone.utc) - timedelta(days=1)).isoformat()
+        self.now = datetime.now(tz=UTC).replace(microsecond=0).isoformat()
+        self.just_now = (datetime.now(tz=UTC).replace(microsecond=0) - timedelta(hours=1)).isoformat()
+        self.older = (datetime.now(tz=UTC).replace(microsecond=0) - timedelta(days=1)).isoformat()
 
         self.measurements = [
-            dict(metric_uuid=METRIC_ID, end=self.now, start=self.now, count=dict(status="target_not_met", value="0")),
-            dict(
-                metric_uuid=METRIC_ID,
-                end=self.just_now,
-                start=self.just_now,
-                count=dict(status="target_met", value="10"),
-            ),
-            dict(metric_uuid=METRIC_ID2, end=self.now, start=self.now, count=dict(status="target_not_met", value="0")),
-            dict(
-                metric_uuid=METRIC_ID2,
-                end=self.just_now,
-                start=self.just_now,
-                count=dict(status="target_met", value="10"),
-            ),
+            {
+                "metric_uuid": METRIC_ID,
+                "end": self.now,
```

Comment:
No, no special reason. I checked whether Black can help with this, but apparently not (TLDR; it would cause side-effects in some situations). See https://github.com/psf/black/issues/523.